### DISCUSSION TITLE
Inconsistent argument name in `write`: `txt` vs `text`

### DISCUSSION THREAD

**Discussion by stenci** [py-pdf/fpdf2#853] [General]
There are five methods that add text to the document: four of them (`text`, `cell`, `multi_cell` and `write`) use the `txt` argument and one of them (`write_html`) uses `text`.

It's inconsistent, isn't pretty, fixing it would break everything, sooo... yeah... I just had to say it.

Maybe it is possible to deprecate that `text` in `write_html`, renaming it to `txt` to make it consistent with the other methods, but keep the good old `txt` still working under the hood for backward compatibility?

**Comment by Lucas-C** [comment]
> It's inconsistent, isn't pretty, fixing it would break everything, sooo... yeah... I just had to say it.

Ahaha 😄  Yeah I agree, it's not pretty

> Maybe it is possible to deprecate that `text` in `write_html`, renaming it to `txt` to make it consistent with the other methods, but keep the good old `txt` still working under the hood for backward compatibility?

Honestly, I'm not convinced that it would be worth the hassle 😅

**Comment by stenci** [comment]
> Honestly, I'm not convinced that it would be worth the hassle 😅

I totally agree! _(that's why this is a discussion and not an issue)_
But I also couldn't keep it for myself 😃

**Comment by Lucas-C** [comment]
For your information, we are planning to deprecate `txt=` in favour of `text=`: #903
